lib/project-publisher.js

```diff
@@ -86,6 +86,9 @@ function uploadFiles()
         
             for(let file of fileList){
                 let relativeFilePath = path.relative(distributionDirPath, file)
+                if (/^win/.test(process.platform)) {
```

Comment:
I will just add a comment above that says windows compatible

Thanks for this PR!